Rakefile

```diff
@@ -1,28 +1,24 @@
-require 'sequel'
+#require 'sequel'
+require 'rake'
+require 'rspec/core/rake_task'
+require File.expand_path '../environment.rb', __FILE__
```

Comment:
Probably, but that was outside the scope of the pr, this is just trying to get an rspec framework into develop